ui/common/InfoBox.tsx

```diff
@@ -0,0 +1,29 @@
+'use client'
+
+function addCommas(number) {
```

Comment:
Browsers have [a built-in API](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/NumberFormat) you can use for this.

Reply:
thanks for the info about this built-in API, I will add it to the code